testsrc/jstests/es6/string.js

```diff
@@ -22,4 +22,8 @@ load("testsrc/assert.js");
   assertEquals(2, "abcdEfg".search("cd", "not used"));
 })();
 
+(function TestFromCodePoint() {
+    assertEquals('☃★♲你', String.fromCodePoint(9731, 9733, 9842, 0x2F804))
```

Comment:
maybe we can make this more robust by using the hex value here - otherwise we have to fight with the js file encoding sooner or later